examples/framework/preferences/minimalFootprint/js/loadScripts.js

```diff
@@ -11,59 +11,53 @@ You may obtain a copy of the ECL 2.0 License and BSD License at
 https://github.com/fluid-project/infusion/raw/main/Infusion-LICENSE.txt
 */
 
-var fluid_4_0_0 = fluid_4_0_0 || {}; // eslint-disable-line no-redeclare
-var fluid = fluid || fluid_4_0_0; // eslint-disable-line no-redeclare
-
-(function (fluid) {
```

Comment:
This file is actually used to load Infusion only when needed. So the `fluid` namespace does not exist yet, and will need to be added for the functions below.